[2023-12-10 19:33:54,117] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.tercera_tarea scheduled__2022-11-07T00:00:00+00:00 [queued]>
[2023-12-10 19:33:54,230] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.tercera_tarea scheduled__2022-11-07T00:00:00+00:00 [queued]>
[2023-12-10 19:33:54,231] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:33:54,640] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-10 19:33:54,647] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:33:55,310] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): tercera_tarea> on 2022-11-07 00:00:00+00:00
[2023-12-10 19:33:55,516] {standard_task_runner.py:52} INFO - Started process 1190 to run task
[2023-12-10 19:33:55,572] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'tercera_tarea', 'scheduled__2022-11-07T00:00:00+00:00', '--job-id', '297', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmpdpegpqxv', '--error-file', '/tmp/tmpqivril96']
[2023-12-10 19:33:55,681] {standard_task_runner.py:80} INFO - Job 297: Subtask tercera_tarea
[2023-12-10 19:33:57,926] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.tercera_tarea scheduled__2022-11-07T00:00:00+00:00 [running]> on host 42828894a739
[2023-12-10 19:33:59,098] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=tercera_tarea
AIRFLOW_CTX_EXECUTION_DATE=2022-11-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-07T00:00:00+00:00
[2023-12-10 19:33:59,321] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-12-10 19:33:59,548] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 3 y sere corrida luego de Tarea 1 al mismo tiempo que Tarea 2']
[2023-12-10 19:33:59,853] {subprocess.py:85} INFO - Output:
[2023-12-10 19:34:00,003] {subprocess.py:92} INFO - hola, soy la tarea 3 y sere corrida luego de Tarea 1 al mismo tiempo que Tarea 2
[2023-12-10 19:34:00,133] {subprocess.py:96} INFO - Command exited with return code 0
[2023-12-10 19:34:00,798] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=tercera_tarea, execution_date=20221107T000000, start_date=20231210T193354, end_date=20231210T193400
[2023-12-10 19:34:01,022] {local_task_job.py:221} WARNING - State of this instance has been externally set to success. Terminating instance.
